spec/models/miq_product_feature_spec.rb

```diff
@@ -16,16 +14,34 @@
     )
   end
 
+  it "is properly configured" do
+    everything = YAML.load_file(described_class.feature_yaml)
+    required_keys = [:identifier]
+    possible_keys = required_keys + [:name, :feature_type, :description, :children, :hidden, :protected]
```

Comment:
@imtayadeway I think it's safe to assume this for the purpose of making this test do a sanity test of the seeded features.